src/test/data/JsonSerializableStudentBookTest/typicalStudentsStudentBook.json

```diff
@@ -2,24 +2,388 @@
   "_comment": "StudentBook save file which contains the same Student values as in TypicalStudents#getTypicalStudentBook()",
   "students": [ {
     "name": "Alice Pauline",
-    "matricNum": "A0000000F"
+    "matricNum": "A0000000F",
+    "gradeProfile" : {
+      "grades" : {
+        "MC1" : {
+          "assessment" : "MC1",
+          "isGraded" : true,
+          "marks" : 0,
+          "totalMarks" : 1
```

Comment:
Seems inefficient to store the total marks for each assignment in every student. Perhaps it would be better to store the assignment details separately to avoid duplication of data.